tests/nativeapp/test_utils.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from snowflake.cli.api.secure_path import SecurePath
+from snowflake.cli.plugins.nativeapp.utils import get_first_paragraph_from_markdown_file
+
+
+@pytest.mark.parametrize(
+    "file_content, expected_paragraph",
+    [
+        (
+            """
```

Comment:
Suggest using `dedent` here

Reply:
I tried and wasn't able to make it work smoothly since some of the whitespace is crucial in how the function works (e.g. first paragraph needs to have no whitespace in between), but I turned the multiline strings into multi line-strings to have a similar effect where the file isn't as long, let me know if that works for you or if I massively misunderstood the assignment 😅